src/views/NoticeOfWithdrawal.vue

```diff
@@ -0,0 +1,1012 @@
+      SaveErrorDialog,
+      SbcFeeSummary
+    }
+  })
+export default class NoticeOfWithdrawal extends Mixins(CommonMixin, DateMixin, FilingMixin, ResourceLookupMixin) {
+    // Refs
+    $refs!: {
+      confirm: ConfirmDialogType,
+      certifyRef: Certify
+    }
+
+    @Getter(useConfigurationStore) getAuthWebUrl!: string
+    @Getter(useRootStore) getBusinessEmail!: string
+    @Getter(useBusinessStore) getLegalName!: string
+    @Getter(useConfigurationStore) getPayApiUrl!: string
+    @Getter(useRootStore) getUserInfo!: any
+    @Getter(useRootStore) isRoleStaff!: boolean
+
+    // enum for template
+    readonly FilingCodes = FilingCodes
+
+    // variables for Certify component
+    certifiedBy = ''
+    isCertified = false
+    certifyFormValid = false
+
+    // variables for POA component
+    hasPlanOfArrangement = false
+    hasComeIntoEffect = false
+
+    // variable for Reference Number component
+    referenceNumberValid = true
+    referenceNumber = ''
+
+    // variables for Document Delivery component
+    documentDeliveryValid = true
+    documentOptionalEmail = ''
+
+    // variables for staff payment
+    staffPaymentValid = false
+    staffPaymentData = { option: StaffPaymentOptions.NONE } as StaffPaymentIF
+
+    // variables for displaying dialogs
+    resumeErrorDialog = false
+    saveErrorReason = null as SaveErrorReasons
+    paymentErrorDialog = false
+
+    // other variables
+    totalFee = 0
+    dataLoaded = false
+    loadingMessage = ''
+    filingId = 0 // id of this Notice if Withdrawal filing
+    savedFiling: any = null // filing during save
+    saving = false // true only when saving
+    savingResuming = false // true only when saving and resuming
+    showErrors = false // true when we press on File and Pay (trigger validation)
+    filingPaying = false // true only when filing and paying
+    haveChanges = false
+    saveErrors = []
+    saveWarnings = []
+    filingToBeWithdrawn = this.$route.params.filingToBeWithdrawn
+
+    /** True if loading container should be shown, else False. */
+    get showLoadingContainer (): boolean {
+      // show loading container when data isn't yet loaded and when
+      // no dialogs are displayed (otherwise dialogs may be hidden)
+      return (!this.dataLoaded && !this.saveErrorReason && !this.paymentErrorDialog)
+    }
+
+    /** The Base URL string. */
+    get baseUrl (): string {
+      return sessionStorage.getItem('BASE_URL')
+    }
+
+    /** True if page is valid, else False. */
+    get isPageValid (): boolean {
+      return (
+        this.certifyFormValid &&
+        this.documentDeliveryValid &&
+        this.referenceNumberValid &&
+        this.staffPaymentValid
+      )
+    }
+
+    /** True when saving, saving and resuming, or filing and paying. */
+    get busySaving (): boolean {
+      return (this.saving || this.savingResuming || this.filingPaying)
+    }
+
+    /** Called when component is created. */
+    created (): void {
```

Comment:
Please add a check for staff either here or in mounted().

A browser alert should suffice if the user is not staff. There may be other examples like this in here or in Edit UI.

Reply:
Example: https://github.com/bcgov/business-edit-ui/blob/f830cd613f410af4a9ce28c89a1da4418c5ff4cf/src/views/Correction.vue#L81

Or do something better 😄

Reply:
updated, how about this? Line 426-429

Reply:
That should be good. Did you test it? What does it look like and what happens next?

Reply:
![image](https://github.com/user-attachments/assets/0ecce5d2-9404-46b9-943e-69b717451b3a)
 And it goes back to the dashboard.

Reply:
The experience looks good, but:
- the title is wrong (I'm starting, not resuming)
- it doesn't tell me that I cannot do this because I'm not staff; in other words, this isn't an "error" -- it's intentional

@jacqueline-williams-549 Thoughts?